File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    bool acceptable = false;
    int input;
    while(!acceptable)
    {
        input = get_int("How many rows? ");
        if(input>0)
        {
            acceptable=true;
        }
    }
    for (int i=1; i<=input; i++) //starts at 1 for easier use of i
    {
        for (int j=0; j<(input-i); j++) //itial gap
        {
            printf(" ");
        }
        for (int j=0; j<i; j++) //hash 1
        {
            printf("#");
        }
        printf("  ");
        for (int j=0; j<i; j++) //hash 2 -- same as 1
        {
            printf("#");
        }
        printf("\n");
    }
}

File: sentimental-mario-more/mario.py
# TODO
from cs50 import get_int
while(True):
    rows = get_int("Height: ")
    if rows>0 and rows<9:
        break

for i in range(rows):
    print(" "*(rows-i-1), end="")
    print("#"*(i+1), end="")
    print("  ", end="")
    print("#"*(i+1))